app/src/main/res/values/strings.xml

```diff
@@ -28,10 +28,10 @@ limitations under the License.
         <item>Date Last Run</item>
     </array>
     <string name="hello_blank_fragment"> About Workflow </string>
-    <string name="Logininfo">Login to your Account (MyExperiment)</string>
+    <string name="LoginInfo">MyExperiment Login</string>
     <string name="loginusername">Email or Username</string>
     <string name="loginpassword">Password</string>
-    <string name="loginbutton">Login</string>
+    <string name="bLogin">Login</string>
```

Comment:
Do not use capital letters here. change to lower case and do not use ambiguous naming